src/helpers/normalizeColor.ts

```diff
@@ -0,0 +1,367 @@
+/*
```

Comment:
No need to review here, but worth taking a look at all the cool strategies for normalizing colors that `normalize-css-color` uses. Lots of bitwise operators!